config/settings.yml.example

```diff
@@ -30,7 +30,9 @@
 # Uncomment and modify if you want to change the default pid file '/var/run/foreman-proxy/foreman-proxy.pid'
 #:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
 
-# HTTP ports configuration
+# host and ports configuration
+# host to bind ports to (default: 0.0.0.0 for production, localhost for development)
```

Comment:
Yeah, though I think the point here is that sometimes it *isn't* defaulting to 0.0.0.0?  like sinatra's ignoring https://github.com/lzap/smart-proxy/blob/binding-config-8905/lib/smart_proxy.rb#L42